src/EnergyPlus/HeatBalanceIntRadExchange.cc

```diff
@@ -190,11 +190,15 @@ namespace HeatBalanceIntRadExchange {
         int startEnclosure = 1;
         int endEnclosure = state.dataViewFactor->NumOfRadiantEnclosures;
         if (PartialResimulate) {
-            startEnclosure = endEnclosure = state.dataHeatBal->Zone(ZoneToResimulate).RadiantEnclosureNum;
-            auto const &enclosure(state.dataViewFactor->ZoneRadiantInfo(startEnclosure));
-            for (int i : enclosure.SurfacePtr) {
-                NetLWRadToSurf(i) = 0.0;
-                state.dataSurface->SurfWinIRfromParentZone(i) = 0.0;
+            // ToDo: For now, use min and max enclosure numbers associated with this zone, this could include unrelated enclosures
+            startEnclosure = state.dataHeatBal->Zone(ZoneToResimulate).zoneRadEnclosureFirst;
+            endEnclosure = state.dataHeatBal->Zone(ZoneToResimulate).zoneRadEnclosureLast;
```

Comment:
And I still need to figure out a better approach for resimulate (primarily used when there's radiant equipment). It's fine now for radiant zones that are entirely in one enclosure. But if a zone is defined with spaces that are not all in the same enclosure, this will calc more than it needs to.